students/will_chang/lesson03/mailroom.py

```diff
@@ -0,0 +1,56 @@
+initial_donor_list = [
+    ["Warren Buffett", 650.00, 2, 325.00],
+    ["Jack Bogle", 15000.50, 1, 15000.50],
+    ["William Boeing", 450.45, 3, 150.15],
+    ["George Clooney", 10000.00, 2, 5000.00],
+    ["Orville Wright", 95000000.00, 1, 950000.00]
+]
+
+def list_data(input_tuple):
```

Comment:
is it really passing in a tuple? you're passing in a list here; however, no need to pass anything in per my other comment.